prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -168,14 +169,19 @@ func getReviewers(rc reviewersClient, author string, files []github.PullRequestC
 	reviewers := sets.NewString()
 	leafReviewers := sets.NewString()
 	ownersSeen := sets.NewString()
-	// first build 'reviewers' by taking a unique reviewer from each OWNERS file.
+	// first build 'reviewers' by taking required reviewers and a unique reviewer from each OWNERS file.
 	for _, file := range files {
 		ownersFile := rc.FindReviewersOwnersForFile(file.Filename)
 		if ownersSeen.Has(ownersFile) {
 			continue
 		}
 		ownersSeen.Insert(ownersFile)
-
+		// add required reviewers if any
+		reviewers.Insert(rc.RequiredReviewers(file.Filename).UnsortedList()...)
+		if reviewers.Len() >= minReviewers {
+			break
+		}
```

Comment:
I am not sure this is the best approach, especially since handle() will slice the list of reviewers to maxReviewers...